### PR TITLE
Additions about languages

### PR DESCRIPTION
**Pull request by milotype**
File update:

- `downloads.md` with some new text and links to the newly added files below.

File additions:

- `gui-locale-available.md` shows a static table of available languages. This could/should be automized in future.
- `gui-locale-how-to.md` explanation for using other languages. I only know about MacOS, other platforms could be done by someone else.

I don't know how to check such changes I did, so maybe someone could check all of the above changes. Thank you.

### CODE REVIEW

File: en-US/downloads/gui-locale-available.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+published: true
+layout: default
+title: Available GUI languages for FontForge
+---
+
+Available GUI languages for FontForge
+--------------
+
+<table class="table" border>
```
Comment (jtanx) [inline en-US/downloads/gui-locale-available.md]: You can write this table in markdown format using somehting like https://www.tablesgenerator.com/markdown_tables

**Comment by jrbastien** [comment]
I will need to review this once I have a computer. This info should already be there under the section "Fontforge, the way you like it".

Envoyé à partir de Yahoo Courriel sur Android 
 
  Le mer., juill. 31 2019 à 9:01 a.m., Jeremy Tan<notifications@github.com> a écrit :   
@jtanx commented on this pull request.

In en-US/downloads/gui-locale-available.md:
> @@ -0,0 +1,117 @@
+---
+published: true
+layout: default
+title: Available GUI languages for FontForge
+---
+
+Available GUI languages for FontForge
+--------------
+
+<table class="table" border>

You can write this table in markdown format using somehting like https://www.tablesgenerator.com/markdown_tables

File: en-US/downloads/gui-locale-available.md (hunk 1 above)
Comment (milotype) [reply en-US/downloads/gui-locale-available.md]: OK, I've changed the table into markdown.

**Comment by milotype** [comment]
@jrbastien ... I didn't find anything there ... but I also feel, such information is good to have at the download page upfront, instead of some deeper documentation page.

**Comment by ctrlcctrlv** [comment]
Closing, no need to have this on the Downloads page. Crowdin provides a good overview of our translation status, we don't need to have it on our website in a format that is liable to become outdated.

**Comment by milotype** [comment]
@ctrlcctrlv Hmm, too bad :-(
The main idea was to give (new) users (possible translators) some information about translations of FontForge, NOT developers.

I don't want to insist, but maybe we could add the following text instead, with a one-clik solution.

FontForge is available for <a href="/en-US/downloads/windows/">Windows</a>, <a href="/en-US/downloads/mac/">Mac</a> and <a href="/en-US/downloads/gnulinux/">GNU+Linux</a> and has been translated into several [languages](https://crowdin.com/project/fontforge).

If you agree, I could make the change and send a new PR.

**Pull request closed without merging**